4 implement crud operation endpoints for store

apokiou opened the pull request:
Implement CRUD Operation Endpoints for 
- Store
- Attribute
- BillingDetails
- Menu
- StoreDayHour 
- Image 
- Feedback

apokiou committed:
CRUD STORE

apokiou committed:
Implement CRUD Operation Endpoints for
.Store
.Attribute
.BillingDetails
.Menu
.StoreDayHour
.Image
.Feedback

apokiou committed:
Implement CRUD Operation Endpoints for
.Store
.Attribute
.BillingDetails
.Menu
.StoreDayHour
.Image
.Feedback

spatiszisis commented on .env:
Delete this file

spatiszisis commented on src/main/java/net/trapezokomos/dashboard/data/Attribute.java:
Remove the unnecessary lines **37-41**
Also the **18 line**
Add space between the **AbstractEntity{**
Change this **private int store_id;**  --> **private int storeId;**

spatiszisis commented on src/main/java/net/trapezokomos/dashboard/data/BillingDetails.java:
Just change this this private int store_id; --> private int **storeId**;

spatiszisis commented on src/main/java/net/trapezokomos/dashboard/data/Feedback.java:
Remove the lines
Also forget to add the lombok annotations to the field e.g Column(...)

spatiszisis commented on src/main/java/net/trapezokomos/dashboard/generator/DataInitializer.java:
Commend all the **createImage** function because we do not insert any data in the database, is unnecessary for now.

spatiszisis commented on src/main/java/net/trapezokomos/dashboard/repository/StoreDayHourRepository.java:
Remove the lines

spatiszisis commented on src/main/java/net/trapezokomos/dashboard/resources/StoreDayHourResource.java:
Remove the unnecessary lines

spatiszisis commented on src/main/java/net/trapezokomos/dashboard/resources/StoreResource.java:
The same here for the lines

spatiszisis commented on src/main/java/net/trapezokomos/dashboard/service/AttributeService.java:
Replace the RuntimeException with the **GenericRuntimeException** and follow the same thing to all the service files

spatiszisis reviewed (changes requested):
Good job overall!
Just see the comments and also pass very quick all the files again for some minor changes 
e.g BaseResource{ --> **BaseResource {**
or some unnecessary lines that I don't mention in the comments

apokiou committed:
#4-Implement CRUD Operation Endpoints for Store, Attribute, BillingDetails, Menu, StoreDayHour, Image, and Feedback Entities
-Delete .env file
-add lombok annotations to the fields in Feedback.java
-Remove the unnecessary lines
-Replace the  RuntimeException with the GenericRuntimeException to all service files

spatiszisis reviewed (approved):
Good!
